plugins/vendor/zoom/infra/kZoomClient.php

```diff
@@ -199,6 +199,10 @@ protected function handleCurlResponse(&$response, $httpCode, $curlWrapper, $apiP
 		{
 			$errMsg = "Zoom Curl returned error, Error code : $httpCode, Error: {$curlWrapper->getError()} ";
 			KalturaLog ::debug($errMsg);
+			if($response)
```

Comment:
need to make sure the zoom response contains the error, else you will need $curlWrapper -> getError()